pos-print/src/main/java/com/clouway/pos/print/core/ReceiptPrintingListener.kt

```diff
@@ -0,0 +1,16 @@
+package com.clouway.pos.print.core
+
+import org.slf4j.LoggerFactory
+
+/**
+ * @author Tsvetozar Bonev (tsvetozar.bonev@clouway.com)
+ */
+class ReceiptPrintingListener : PrintingListener {
+  override fun onPrinted(receipt: Receipt, printStatus: PrintStatus) {
+    val logger = LoggerFactory.getLogger(PrintingListener::class.java)
```

Comment:
It would be better if you bring the initialization out of the function

Reply:
Must've overlooked this, moved it into the class.